### PR TITLE
New Kinetis board support

### PR DESCRIPTION
**Pull request by flit**
Tested targets for which I have boards. Other targets were tested by other people internally.

Sorted the board list by ID.

### CODE REVIEW

File: pyOCD/flash/flash_k82f25615.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+"""
```
Comment (c1728p9) [inline pyOCD/flash/flash_k82f25615.py]: I didn't see an algo for this guy in the FlashAlgo repo. Could you add one for this?

**Comment by c1728p9** [comment]
Thanks for the PR. It looks good to me.

**Comment by c1728p9** [comment]
@mbed-bot: TEST

File: pyOCD/flash/flash_k82f25615.py (hunk 1 above)
Comment (flit) [inline pyOCD/flash/flash_k82f25615.py]: It's the same as the K80 that's already in FlashAlgo.

**Comment by flit** [comment]
Rebased to resolve conflict.

**Comment by c1728p9** [comment]
@mbed-bot: TEST

**Comment by c1728p9** [comment]
Thanks for the new targets

**Pull request merged**